From b8318ed89ba2bddc8305465e39614f75c1bf7852 Mon Sep 17 00:00:00 2001
From: Lin Huang <hl@rock-chips.com>
Date: Sun, 7 Oct 2018 16:27:57 +0800
Subject: [PATCH] arm64: rk1808_linux_defconfig: build AP6212 wifi driver as
 module

buildroot use rkwifibt to install wifi driver, and it request wifi
driver build as module.

Change-Id: Iad26078abe4b02f55def46f4c9f84b3d7f2d7724
Signed-off-by: Lin Huang <hl@rock-chips.com>
---
 arch/arm64/configs/rk1808_linux_defconfig | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/arch/arm64/configs/rk1808_linux_defconfig b/arch/arm64/configs/rk1808_linux_defconfig
index 89ee3eb3073c..cb4b9fbb6245 100644
--- a/arch/arm64/configs/rk1808_linux_defconfig
+++ b/arch/arm64/configs/rk1808_linux_defconfig
@@ -98,8 +98,9 @@ CONFIG_NETDEVICES=y
 # CONFIG_NET_CORE is not set
 # CONFIG_ETHERNET is not set
 CONFIG_WL_ROCKCHIP=y
+CONFIG_WIFI_BUILD_MODULE=y
 CONFIG_WIFI_LOAD_DRIVER_WHEN_KERNEL_BOOTUP=y
-CONFIG_AP6XXX=y
+CONFIG_AP6XXX=m
 # CONFIG_RTL_WIRELESS_SOLUTION is not set
 # CONFIG_INPUT_MOUSEDEV is not set
 CONFIG_INPUT_EVDEV=y
-- 
2.35.3

